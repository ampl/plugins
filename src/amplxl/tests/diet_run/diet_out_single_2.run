
model diet.mod;

param fname_out symbolic := "diet_xlsx/diet_out_single_2.xlsx";

table Amounts IN "amplxl" (fname_out): 
    [NUTR,FOOD], amt;

table Nutrients IN "amplxl" (fname_out):
    NUTR <- [NUTR], n_min, n_max;

table Foods "amplxl" (fname_out):
    [FOOD] IN, cost IN, f_min IN, f_max IN,
    Buy OUT, Buy.rc ~ BuyRC OUT,
    {j in FOOD} Buy[j]/f_max[j] ~ BuyFrac OUT;

read table Foods;
read table Nutrients;
read table Amounts;

display NUTR;
display FOOD;
display amt;

solve;

write table Foods;
