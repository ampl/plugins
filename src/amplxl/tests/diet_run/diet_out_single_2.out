set NUTR := A C B1 B2 NA CAL;

set FOOD := BEEF CHK FISH HAM MCH MTL SPG TUR;

amt [*,*] (tr)
:      A    B1   B2   C    CAL    NA     :=
BEEF   60   10   15   20   295    938
CHK     8   20   20    0   770   2180
FISH    8   15   10   10   440    945
HAM    40   35   10   40   430    278
MCH    15   15   15   35   315   1182
MTL    70   15   15   30   400    896
SPG    25   25   15   50   370   1329
TUR    60   15   10   20   450   1397
;

MINOS 5.51: optimal solution found.
13 iterations, objective 118.0594032
creating diet_xlsx/diet_out_single_2.xlsx
